tmt/steps/provision/artemis.py

```diff
@@ -335,7 +335,7 @@ def _create(self) -> None:
         self.guestname = response.json()['guestname']
         self.info('guestname', self.guestname, 'green')
 
-        with updatable_message(
+        with UpdatableMessage(
```

Comment:
This would mean renaming the tmt.utils.py `class UpdatableMessage(contextlib.AbstractContextManager)` back to lowercase and adding noqa for N801. Do you want me to do that?

Reply:
Yes, please. Unless greater powers step in, I for one prefer `updatable_message` (and `retry_session`) - they did pass their review, and I hate the idea of a possible rename if we decide to rewrite them as functions with `yield` :)

Reply:
Reverted to `updatable_message` and `retry_session`. Thanks!